src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+
+export const Person = (props) => {
+  const { name, age, sex, isMarried, partnerName } = props.person;
+
+  let partner;
+  if (sex === 'm' && isMarried) {
+    partner = 'wife';
+  } else {
+    partner = 'husband';
+  }
+
+  return (
+    <section className="Person">
+      {name &&
+        <h2 className="Person__name">My name is {name}</h2>
+      }
+      {age &&
+        <p className="Person__age">I am {age}</p>
+      }
+      {isMarried
+        ? <p className="Person__partner">{partnerName} is my {partner} </p>
+        : <p className="Person__partner">I am not married</p>
+      }
```

Comment:
```suggestion
      <p className="Person__partner">{personPartner}</p>
```
only test in Person__partner will be different.

Reply:
Thank you!
I got it)